### PR TITLE
Only test vs ctapipe master and ctapipe_io_lst master

### PR DESCRIPTION
**Pull request by MaxNoe**
Update the travis config so that only tested vs the master branch of ctapipe and that it is required to pass

### CODE REVIEW

**Comment by MaxNoe** [comment]
Why doesn't the travis build show up here?

**Comment by MaxNoe** [comment]
> Why changing to a fixed version of protozfits but keeping ctapipe_io_lst master ?

I only removed an unused environment variable. The protozfits dependency is handled by `ctapipe_io_lst` automatically.

**Comment by MaxNoe** [comment]
Ah, sorry, wrong PR, can be removed here as well.

**Comment by MaxNoe** [comment]
Also please note that this PR is expected to fail on travis, as we do not yet support the current ctapipe master.

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,29 @@
 ---
 language: python
 
-python:
-    - 3.6
-    - 3.7
-
 env:
-    - CTAPIPE_VERSION="v0.7.0" CTAPIPE_IO_LST_VERSION="v0.4.1" PROTOZFITS_VERSION=v1.4.2
```
Comment (rlopezcoto) [inline .travis.yml:9]: I have here the same comment as in https://github.com/cta-observatory/ctapipe_io_lst/pull/32, could you add the ctapipe v0.8 test as well?

**Pull request closed without merging**